src/bundles/config.js

```diff
@@ -13,6 +17,15 @@ const bundle = createAsyncResourceBundle({
       conf = JSON.stringify(rawConf, null, '\t')
     }
 
+    // More info: https://github.com/ipfs-shipyard/ipfs-webui/issues/1490#issuecomment-671633602
+    const config = JSON.parse(conf)
+    const url = getURLFromAddress('Gateway', config) || DEFAULT_URI
+
+    if (!await checkIfGatewayUrlIsAccessible(url)) {
+      config.Addresses.AvailableGateway = DEFAULT_MULTI_ADDR
```

Comment:
Persisting `AvailableGateway` in `config` smells :biohazard:  
We are mutating config and then lie to user  on _Settings_ screen:

![2020-08-26--13-43-43](https://user-images.githubusercontent.com/157609/91300472-9c870000-e7a3-11ea-8c2b-8675c958a8ef.png)


My main worry is this will bite us in the future in a way that breaks users node (invalid config is saved, then node fails to start). Let's avoid this line of hacking at all cost.

@rafaelramalho19 can we leave `config` as-is and store `AvailableGateway` in something like `window.localStorage` instead?

Reply:
You're right, I moved it into a separate persistence bundle :)